### PR TITLE
Add Shell scripts

### PR DESCRIPTION
**Pull request by pellared**
Add Shell scripts based on https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -49,6 +49,40 @@ When running your application, make sure to:
 
 > Some settings can be omitted on .NET (Core). For more information, see the [documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.3.1-beta.1/docs/config.md#net-clr-profiler).
 
+### Shell scripts
+
+You can install Splunk Distribution of OpenTelemetry .NET
+and instrument your .NET application using the provided Shell scripts.
+Example usage:
```
Comment (theletterf) [inline docs/README.md:56]:
```suggestion
You can install the Splunk Distribution of OpenTelemetry .NET
and instrument your .NET application using the official shell scripts.

For example:
```

File: docs/README.md [hunk 2]
```diff
@@ -49,6 +49,40 @@ When running your application, make sure to:
...
+
+```sh
+curl -sSfL https://raw.githubusercontent.com/signalfx/splunk-otel-dotnet/v0.0.1-alpha.1/splunk-otel-dotnet-install.sh -O
+sh ./splunk-otel-dotnet-install.sh
+. $HOME/.splunk-otel-dotnet/instrument.sh
+OTEL_SERVICE_NAME=myapp OTEL_RESOURCE_ATTRIBUTES=deployment.environment=staging,service.version=1.0.0 dotnet run
+```
+
+[splunk-otel-dotnet-install.sh](../splunk-otel-dotnet-install.sh) script
```
Comment (theletterf) [inline docs/README.md:65]:
```suggestion
The [splunk-otel-dotnet-install.sh](../splunk-otel-dotnet-install.sh) script
```

File: docs/README.md [hunk 3]
```diff
@@ -49,6 +49,40 @@ When running your application, make sure to:
...
+uses environment variables as parameters:
+
+| Parameter               | Description                                                      | Required | Default value             |
+|-------------------------|------------------------------------------------------------------|----------|---------------------------|
+| `OTEL_DOTNET_AUTO_HOME` | Location where binaries are to be installed                      | No       | `$HOME/.splunk-otel-dotnet` |
+| `OS_TYPE`               | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows` | No       | *Calculated*              |
+| `TMPDIR`                | Temporary directory used when downloading the files              | No       | `$(mktemp -d)`            |
+| `VERSION`               | Version to download                                              | No       | `v0.0.1-alpha.1`           |
+
+[instrument.sh](../instrument.sh) script
```
Comment (theletterf) [inline docs/README.md:75]:
```suggestion
The [instrument.sh](../instrument.sh) script
```

File: docs/README.md (hunk 1 above)
Comment (theletterf) [reply docs/README.md:56]: Can we explain the difference between the install and instrument scripts?

**Review by theletterf — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (pellared) [reply docs/README.md]: How about:

```
The [splunk-otel-dotnet-install.sh](../splunk-otel-dotnet-install.sh) script
installs the OpenTelemetry .NET Automatic Instrumentation.
It uses environment variables as parameters:
```

```
The [instrument.sh](../instrument.sh) script
enables the instrumentation in the current shell session.
It uses environment variables as parameters:
```

I am not sure how to make it clear that "install" only "places the component on local machine".

**Review by Kielek — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (theletterf) [reply docs/README.md]: So instrument includes install ? Or they're just separate steps to be run in sequence? If that's the case, what's the reason for splitting them?

Comment (pellared) [reply docs/README.md]: Will be addressed in a separate PR

Comment (pellared) [reply docs/README.md]: https://github.com/signalfx/splunk-otel-dotnet/pull/42

**Pull request merged**